### PR TITLE
Fixed a few things with the HF blade (hopefully)

### PR DESCRIPTION
**Pull request by xDyouGamer**
## About The Pull Request
-Only able to play music in brazil mode
-Cooldown between the music so you cant spam it like an idiot
-Pop req for the blade is now 20 to prevent cringe murderbone

## Why It's Good For The Game
no idea

## Changelog
:cl:
add: Fixes the HF blade
/:cl:

### CODE REVIEW

**Comment by ro5490** [comment]
@xDyouGamer 
image

File: code/game/objects/items/melee/hfblade.dm [hunk 1]
```diff
@@ -72,13 +72,24 @@
 		user.changeNext_move(CLICK_CD_CLICK_ABILITY)
 		return ..()
 
+
+var/cooldown = 40 // 4 seconds
+var/last_used = 0
```
Comment (MACIEKBAKI) [inline code/game/objects/items/melee/hfblade.dm:77]: remove this

Comment (MACIEKBAKI) [reply code/game/objects/items/melee/hfblade.dm:77]: and put it near the top where all the other vars are defined

Comment (MACIEKBAKI) [reply code/game/objects/items/melee/hfblade.dm:77]:
```
	light_color = "#40ceff"
	var/rules_of_nature = TRUE // Turn this off to break the rules, and watch the horrors of no proximity flags unfold.
	var/brazil = FALSE
	var/brightness = 5
	var/cooldown = 40
	var/last_used = 0
```

is what you want

**Review by MACIEKBAKI — CHANGES_REQUESTED** [review]

**Review by ro5490 — APPROVED** [review]

**Pull request merged**